src/javascript/JContent/ContentRoute/ContentLayout/PreviewDrawer/Preview/PreviewComponent/DocumentViewer/DocumentViewer.scss

```
@@ -10,7 +10,8 @@
 
 .container.container {
     flex-grow: 1;
-    max-height: 100%;
+    width: 100%;
```

I think you also have to add this to center the icon when we can't display a preview (`.doc` files for example).
```suggestion
    width: 100%;
    display: flex;
    align-items: center;
    justify-content: center;
```